File: npm-compiler/codes/Execute.py

import time;

st = time.time();


n = 10;  
m = (2 * n) - 2  
for i in range(0, n):  
    for j in range(0, m):  
        print(end=" ")  
    m = m - 1 
    for j in range(0, i + 1):  
        
        print("* ", end=' ')  
    print(" ") 


et = time.time();
elapsed_time = et - st;
print('Execution Time :',round(elapsed_time,3), 's');

File: npm-compiler/codes/Execute.java
import java.util.concurrent.TimeUnit;
import java.io.*;
class UserName1
{
	public static void printStars(int n)
	{
		int i, j;
		for(i=0; i<n; i++)
		{
			for(j=0; j<=i; j++)
			{
				System.out.print("* ");
			}
			System.out.println();
		}
}


	public static void main(String args[])
	{
		int n = 5;
		printStars(n);
	}
}



public class Execute {
	public static void main(String args[]) {
		long startTime = System.nanoTime();

		UserName1.main(args);

		long stopTime = System.nanoTime();
		long elapsedTime = (stopTime - startTime);
		System.out.println("Execution Time :"+(TimeUnit.MILLISECONDS.convert(elapsedTime, TimeUnit.NANOSECONDS) / 1000.0)+"s");
	}
}
